src/openvic-simulation/scripts/Condition.cpp

```diff
@@ -721,7 +721,7 @@ node_callback_t ConditionManager::expect_condition_node_list(
 		};
 
 		bool ret = conditions.expect_item_dictionary_and_default(
-			expect_node, top_scope ? top_scope_fallback : key_value_invalid_callback
+			top_scope ? top_scope_fallback : key_value_invalid_callback, expect_node
```

Comment:
Are you sure this is intentional and not some leftover code?
I thought we just got conditions working.

Reply:
I switched the order of arguments in `expect_item_dictionary_and_default` to match `expect_dictionary_keys_and_default`.